repo-projects/Directory.Build.targets

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
 
   <UsingTask TaskName="Microsoft.DotNet.Arcade.Sdk.SourceBuild.AddSourceToNuGetConfig" AssemblyFile="$(ArcadeSdkBuildTasksAssembly)" />
+  <UsingTask TaskName="UpdateJson" AssemblyFile="$(XPlatSourceBuildTasksAssembly)" />
```

Comment:
Where does this task get called from?

Reply:
The consuming target already existed but failed when invoked...

https://github.com/dotnet/source-build-externals/pull/434/files#diff-c4c7386806d2707bdf65085bbf28501450a36181d035aa9a306cae52afd6f2f0L124